File: Vohra_ProMongoDB_SourceCode/ch5/scripts/connection.js
MongoClient = require('mongodb').MongoClient;

var mongoclient = new MongoClient();
 

mongoclient.connect("mongodb://localhost:27017/test", function(error, db) {
     if (error) 
         console.log(error);
     else 
         console.log('Connected with MongoDB');
  });
  

File: Vohra_ProMongoDB_SourceCode/ch4/mongodbscripts/connection.rb
require 'mongo'
include Mongo

 
#client =Mongo::Client.new([ '127.0.0.1:27017' ], :database => 'test')
#client =Mongo::Client.new([ '127.0.0.1:27017' ], :database => 'test', :connect => :direct)
#client =Mongo::Client.new('mongodb://127.0.0.1:27017/test')
client = Mongo::Client.new([ '127.0.0.1:27017' ], :database => 'test', :max_pool_size => 20, :connection_timeout => 60)

print "Cluster: "
print  client.cluster
print "\n"
 
print "Configuration Options: "
print  client.options
print "\n"  
print "Read Preference: "
print  client.read_preference
print "\n" 
print "Write Concern : "
print client.write_concern
print "\n"  
   